src/webWorker/webWorker.js

```diff
@@ -0,0 +1,142 @@
+/*eslint-disable*/
+// an object of task handlers
+const taskHandlers = {}
+
+// Flag to ensure web worker is only initialized once
+let initialized = false
+
+// the configuration object passed in when the web worker manager is initialized
+let config
+
+/**
+ * Initialization function that loads additional web workers and initializes them
+ * @param data
```

Comment:
It's currently not very clear what data is passed around via the different messages.

In general, I prefer explicitly specifying the attributes of parameter/option objects (e.g,. `{config, workerIndex}` instead of just `data`).

Reply:
done in https://github.com/herrmannlab/dicom-microscopy-viewer/pull/76/commits/7b34d33d4353c4ad02e06479da4e37ea3f68fc10